ADR01_application_structure.md

```diff
@@ -0,0 +1,61 @@
+# Application structure
+
+Contents:
+
+* [Summary](#summary)
+  * [Issue](#issue)
+  * [Decision](#decision)
+  * [Status](#status)
+* [Details](#details)
+  * [Assumptions](#assumptions)
+  * [Constraints](#constraints)
+  * [Alternatives](#alternatives)
+  * [Argument](#argument)
+  * [Justification](#justification)
+
+
+## Summary
+
+
+### Issue
+
+We want to use an architecture that enables us to have maintainable software, as well to be expandable easily and can use different supporting tools and plugins in the different software layers based on different technologies.
+  
+### Decision
+
+The application should be based on Model, View, and Controller (A three tieres architecture)
```

Comment:
Well, MVC is kind of old. It turned out that is also a bit harder to maintain compared to N-Layered Architecture or Onion Architecture